app/src/main/java/com/sunlightlabs/android/congress/fragments/BillActionFragment.java

```diff
@@ -104,14 +104,14 @@ public BillActionAdapter(ListFragment context) {
 
 		@Override
 		public Date dateFor(Bill.Action action) {
-			return action.acted_at;
+			return action.acted_on;
```

Comment:
Most fields went from datetimes to just dates, so I renamed fields from `_at` to `_on` where appropriate to signify that.